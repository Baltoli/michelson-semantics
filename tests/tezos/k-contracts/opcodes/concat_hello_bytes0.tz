$contract { parameter (list bytes);
storage (list bytes);
code{ CAR;
      MAP { PUSH bytes 0xFF; CONCAT }; NIL operation; PAIR};
 } ;
$param { 0xcd } ;
$storage {} ;
$ExpectedOutput { 0xffcd } ;

