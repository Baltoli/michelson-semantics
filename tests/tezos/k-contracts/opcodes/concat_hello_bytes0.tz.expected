<generatedTop>
  <k>
    #VerifyStorage ( ListItem ( 0xffcd ) )
  </k>
  <stack>
    Pair .List ListItem ( #UnknownMBytes )
  </stack>
  <paramtype>
    list .AnnotationList bytes .AnnotationList
  </paramtype>
  <storagetype>
    list .AnnotationList bytes .AnnotationList
  </storagetype>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( "InvalidMyAddr" )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( "InvalidSourceAddr" )
  </sourceaddr>
  <senderaddr>
    #Address ( "InvalidSenderAddr" )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
